tests/sls/common/timezone.py

```diff
@@ -0,0 +1,18 @@
+import subprocess
+
+from tests.util import Failure, Success
+
+
+def run():
+    command = "date | grep -v UTC"
```

Comment:
Instead of using pipes and an extra `grep` process, let's just look for `UTC` in the output of the `date` command inside Python.